src/atoms/filledArrow/FilledArrowBtn.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+import Button from "../button/Button";
+import Icon from "../icon/Icon";
+
+interface FilledArrowBtnProps {
+  isPrimary: boolean;
+  direction: HorizontalDirection;
+  width?: React.CSSProperties["width"];
+  height?: React.CSSProperties["height"];
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export enum HorizontalDirection {
+  LEFT = "left",
+  RIGHT = "right",
+}
+
+const AnimatedBtn = styled(Button)`
+  transition: transform 0.1s ease-in-out;
+  :active {
+    transform: scale(0.9);
+  }
+`;
+
+const FilledArrowBtn = ({
+  isPrimary,
+  direction,
+  width,
+  height,
+  onClick,
+}: FilledArrowBtnProps) => {
+  switch (direction) {
+    case HorizontalDirection.LEFT:
+      if (isPrimary) {
+        // Left Primary
+        return (
+          <AnimatedBtn onClick={onClick}>
+            <Icon.ArrowCircleLeftPrimaryFilled width={width} height={height} />
+          </AnimatedBtn>
+        );
+      }
+      // Left Deselected
+      return (
+        <AnimatedBtn onClick={onClick}>
+          <Icon.ArrowCircleLeftDeselectedFilled width={width} height={height} />
+        </AnimatedBtn>
+      );
+    case HorizontalDirection.RIGHT:
+      if (isPrimary) {
+        // Right Primary
+        return (
+          <AnimatedBtn onClick={onClick}>
+            <Icon.ArrowCircleRightPrimaryFilled width={width} height={height} />
+          </AnimatedBtn>
+        );
+      }
+      // Right Deselected
+      return (
+        <AnimatedBtn onClick={onClick}>
+          <Icon.ArrowCircleRightDeselectedFilled
+            width={width}
+            height={height}
+          />
+        </AnimatedBtn>
+      );
+  }
```

Comment:
switch문 좋은데 AnimatedBtn 안쪽으로 집어넣으면 더 깔끔할 것 같아요
```suggestion

  return (
    <AnimatedBtn onClick={onClick}>
    {
      (() => {
        switch (direction) {
          case HorizontalDirection.LEFT:
              // Left
              return isPrimary ? <Icon.ArrowCircleLeftPrimaryFilled width={width} height={height} /> : <Icon.ArrowCircleLeftDeselectedFilled width={width} height={height} />
              
                case HorizontalDirection.RIGHT:
                   return isPrimary ? <Icon.ArrowCircleRightPrimaryFilled width={width} height={height} /> ? <Icon.ArrowCircleRightDeselectedFilled width={width} height={height} />
          } 
       })()
    }
    </AnimatedBtn>
  )
```